src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -136,9 +151,9 @@ const init = ({
     },
     minimumPower: 750,
     query: '',
-    statOrder: statKeys,
+    statOrder: settingsStatOrder.map((hash) => statHashToType[hash]),
     selectedStoreId: getCurrentStore(stores)?.id,
-    assumeMasterwork: false,
+    assumeMasterwork: settingsAssumeMasterwork,
```

Comment:
For some reason settingsAssumeMasterwork is undefined after you reload the page but settingsStatOrder has the correct value saved. I can't figure out why, do you have any hunches?